torch/optim/_multi_tensor/__init__.py

```diff
@@ -6,7 +6,19 @@
 """
 from functools import partialmethod
 
-from torch import optim
+from .. import (
+    Adadelta as _Adadelta,
+    Adagrad as _Adagrad,
+    Adam as _Adam,
+    Adamax as _Adamax,
+    AdamW as _AdamW,
+    ASGD as _ASGD,
+    NAdam as _NAdam,
+    RAdam as _RAdam,
+    RMSprop as _RMSprop,
+    Rprop as _Rprop,
+    SGD as _SGD,
+)
```

Comment:
```suggestion
import torch
```

Use `torch.optim.XXX` below.

Reply:
Circular imports with absolute paths are causing failures in our environment (infinite recursion). Using relative paths fixes that.